hw/bsp/same54xplainedpro/same54xplainedpro.c

```diff
@@ -166,15 +166,16 @@ void board_init(void)
 #endif
 
 	// LED0 init
+	gpio_set_pin_function(BUTTON_PIN, GPIO_PIN_FUNCTION_OFF);
 	gpio_set_pin_direction(LED_PIN, GPIO_DIRECTION_OUT);
-	gpio_set_pin_level(LED_PIN, 0);
+	gpio_set_pin_level(LED_PIN, 1);
```

Comment:
How about `board_led_write(0)` to set led to off

Reply:
Done